Makefile

```diff
@@ -10,4 +10,4 @@ localdeps:
 	pip install --user --requirement requirements.txt
 
 test:
-	nosetests
+	pytest --disable-pytest-warnings
```

Comment:
Seems like `--disable-pytest-warnings` is just an alias to `--disable-warnings` at least according to the output from `pytest --help`. `--disable-warnings` is also the one mentioned in [the documentation](https://docs.pytest.org/en/latest/warnings.html#disabling-warnings-summary), so I think we can switch to that one instead.

Reply:
7364208